common/src/main/scala/com/aliyun/emr/rss/common/protocol/message/ControlMessages.scala

```diff
@@ -319,7 +319,30 @@ object ControlMessages extends Logging {
 
   case class ThreadDumpResponse(threadDump: String) extends Message
 
-  case class UserIdentifier(tenantId: String, name: String) extends Message
+  case class UserIdentifier(tenantId: String, name: String) extends Message {
+    assert(
```

Comment:
After the regex check procedures, the tenant id and name can not be empty. So I think the assertions are not necessary.